sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj

```diff
@@ -10,11 +10,13 @@
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Network" (https://www.nuget.org/packages/Azure.ResourceManager.Network/). Please see the package changelog for more information.
 		</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>20.0.4-preview</Version>
+		<Version>20.1.0-preview</Version>
```

Comment:
I see the swagger is in stable version now, do you still need -preview?

Reply:
No, can I simply call it 20.1.0?